pkg/shell/cockpit-networking.js

```diff
@@ -3091,21 +3094,22 @@ PageNetworkVlanSettings.prototype = {
                 parent_choices.push({ title: i.Name, choice: i.Name });
         });
 
+        /* TODO: Migrate away from jQuery spaghetti */
```

Comment:
possibly add comment about adding linking code between labels and input controls when this is cleaned up